From df7ddf004d4613c247db81c6fde87633037dfbef Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@streambits.io>
Date: Mon, 8 Feb 2021 20:32:43 +0200
Subject: [PATCH 1/1] Rename obs-browser-page directory to
 obs-browser-page-main. Closes #239

As discussed in #239, `cmake` is being instructed to build the `obs-browser` binary using the same name as a directory that already exists.

This pull request renames the `obs-browser-page` directory to `obs-browser-page-main` and updates `CMakeLists.txt` accordingly.

This fixes the FTBFS report in #239 using Ubuntu 20.04, `gcc` 9.3.0 and `cmake` 3.16.

While I did not encounter this issue using Ubuntu 18.04, `gcc` 8.4.0 and ``cmake` 3.10; I've tested building on that platform with this fix applied and compilation continues to complete successfully.

Original patch from: Martin Wimpress <code@flexion.org>
---
 CMakeLists.txt                                                | 4 ++--
 .../obs-browser-page-main.cpp                                 | 0
 2 files changed, 2 insertions(+), 2 deletions(-)
 rename {obs-browser-page => obs-browser-page-main}/obs-browser-page-main.cpp (100%)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9a8508a..b66d1f5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -249,12 +249,12 @@ if (APPLE AND NOT BROWSER_LEGACY)
   	endforeach()
 else()
     set(obs-browser-page_SOURCES
-    	obs-browser-page/obs-browser-page-main.cpp
+    	obs-browser-page-main/obs-browser-page-main.cpp
     	browser-app.cpp
     	deps/json11/json11.cpp
     	)
     set(obs-browser-page_HEADERS
-    	obs-browser-page/obs-browser-page-main.cpp
+    	obs-browser-page-main/obs-browser-page-main.cpp
     	browser-app.hpp
     	deps/json11/json11.hpp
     	cef-headers.hpp
diff --git a/obs-browser-page/obs-browser-page-main.cpp b/obs-browser-page-main/obs-browser-page-main.cpp
similarity index 100%
rename from obs-browser-page/obs-browser-page-main.cpp
rename to obs-browser-page-main/obs-browser-page-main.cpp
-- 
2.30.0

